db/views/annual_energy_costs_v01.sql

```diff
@@ -0,0 +1,55 @@
+WITH electricity AS (
+  SELECT alert_generation_run_id, data.*
+  FROM alerts, alert_types, jsonb_to_record(variables) AS data(
+    last_year_gbp float
+  )
+  WHERE alerts.alert_type_id = alert_types.id and alert_types.class_name='AlertElectricityAnnualVersusBenchmark'
+), gas AS (
+  SELECT alert_generation_run_id, data.*
+  FROM alerts, alert_types, jsonb_to_record(variables) AS data(
+    last_year_gbp float
+  )
+  WHERE alerts.alert_type_id = alert_types.id and alert_types.class_name='AlertGasAnnualVersusBenchmark'
+), storage_heaters AS (
+  SELECT alert_generation_run_id, data.*
+  FROM alerts, alert_types, jsonb_to_record(variables) AS data(
+    last_year_gbp float
+  )
+    WHERE alerts.alert_type_id = alert_types.id and alert_types.class_name='AlertStorageHeaterAnnualVersusBenchmark'
+), total AS (
```

Comment:
energy vs total?

Reply:
Yes - you're right, that would be better 👍